spec/models/manageiq/providers/autosde/storage_manager/refresher_spec.rb

```diff
@@ -31,6 +30,64 @@
       end
     end
 
+    context "targeted refresh" do
+      before { VCR.use_cassette("ems_refresh") { described_class.refresh([ems]) } }
+      let(:system_type_api) { double("SystemTypeApi") }
+      let(:storage_system_api) { double("StorageSystemApi") }
+
+      it "with no targets" do
+        assert_inventory_not_changed { run_targeted_refresh }
+      end
+
+      it "with an existing physical_storage target" do
+        expect(storage_system_api)
+          .to receive(:storage_systems_get)
+          .and_return(
+            [
+              AutosdeOpenapiClient::StorageSystem.new(
```

Comment:
This is a duplicate of the payload that is returned for this storage system, the goal here is to run a targeted refresh that will result in 0 changes allowing us to `assert_inventory_not_changed`

We should have one of these for every targeted type that we support, this ensure that nothing is being deleted & recreated while targeting the different collections